src/app/jscaip/Coord.ts

```diff
@@ -103,9 +102,10 @@ export class Coord implements ComparableObject {
         return false;
     }
     public getDirectionToward(c: Coord): Direction {
+        // TODO: method might have to be deleted in favor of Direction.fromMove
         const dx: number = Coord.getBinarised(c.x - this.x);
         const dy: number = Coord.getBinarised(c.y - this.y);
-        return Direction.factory.of(dx, dy); // TODO: method might have to be deleted in favor of Direction.fromMove
+        return Direction.factory.of(dx, dy).get();
```

Comment:
supprimée non mais qu'elle délègue, éventuellement.

Reply:
hmm elle délègue déjà, à .of

Reply:
Je veux dire, déléguer à fromMove
avoir coord.getDirectionToward(c) { return Direction.factory.fromMove(coord, c) }

Reply:
Ça n'irait point car fromMove balance une erreur si les coordonnées sontpas alignées, alors que getDirectionToward non ! (Il me semble en tout cas)

Reply:
Ah bah oui avec le getBinarised ! 
Arh du coup celui ci donne une direction général ! si t'es à 2 à gauche et 5 en bas il dira que t'es en bas à gauche !
EST CE QUE NOTRE CODE SAIS CA :D ????
En tout cas faut soit changer le comportement, soit le nom :o !
Bien vu la vue!

Reply:
Eh ben je croyais qu'il était nécessaire mais me souvenais plus pourquoi, mais déléguer à .fromMove(this, c).get(); ne casse aucun test. Du coup, je propose de changer le type de retour en MGPFallible, soit simplement de remplacer les appels à cette fonction par des appels à Direction.factory? (y'a 7 appels, dans abalone, epaminondas, et tablut, tandis qu'il y en a une quinzaine à Direction.factory.fromMove, étonamment y'en a aussi dans abalone et epaminondas :D).

Et aussi: supprimer getBinarised qui sera non utilisée